tox.ini

```
@@ -5,7 +5,7 @@ envlist =
     {py37,py38}-django30
 
 [testenv]
-commands = python manage.py test {posargs}
+commands = python manage_test.py test {posargs}
```

I believe you don't need another `manage.py`, just setting the correct ENV variable.
```suggestion
commands = env DJANGO_SETTINGS_MODULE="test_project.dp_test_settings" python manage.py test {posargs}
```